datastore/tests/system/test_system.py

```diff
@@ -442,6 +443,17 @@ def test_transaction_via_with_statement(self):
         self.case_entities_to_delete.append(retrieved_entity)
         self.assertEqual(retrieved_entity, entity)
 
+    def test_readwrite_put(self):
+        entity = datastore.Entity(key=Config.CLIENT.key('Company', 'Google'))
+        options = types.TransactionOptions(
+            read_write=types.TransactionOptions.ReadWrite())
+        with Config.CLIENT.transaction() as xact:
+            xact.put(entity)
```

Comment:
You never actually use the `options` variable (PyLint probably complains but maybe `flake8` doesn't).

If you wanted to actually use it you'd need to so `xact.options = options` (after making it non-public `xact._options = options`)